src/Simulation/Simulators/SparseSimulator2/Dump.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Quantum.Simulation.Core;
+
+namespace Microsoft.Quantum.Simulation.Simulators
+{
+    using QubitIdType = System.IntPtr;
+
+    public partial class SparseSimulator2
```

Comment:
Is the name `SparseSimulator2` final? It's a little odd for the first public version of a feature to have "2" in the name.

Reply:
Not final. Once the `SparseSimulator2` is fully integrated, the older `SparseSimulator` will be deleted, and the `SparseSimulator2` will be renamed to `SparseSimulator`. Some more details are in the description of this PR.